merlin/systems/dag/ops/softmax_sampling.py

```diff
@@ -133,6 +133,10 @@ def transform(
         # This is just bookkeeping to produce the final ordered list of recs
         sorted_indices = np.argsort(exponentials)
         topk_item_ids = candidate_ids[sorted_indices][: self.topk]
+        topk_item_scores = predicted_scores[sorted_indices][: self.topk]
         ordered_item_ids = topk_item_ids.reshape(1, -1).T
+        ordered_item_scores = topk_item_scores.reshape(1, -1).T
 
-        return type(transformable)({"ordered_ids": ordered_item_ids})
+        return type(transformable)(
```

Comment:
do we nee to change `compute_output_schema` to match this?

Reply:
Will fix